qlib/data/dataset/loader.py

```diff
@@ -320,7 +321,12 @@ def __init__(self, dataloader_l: List[Dict], join="left") -> None:
     def load(self, instruments=None, start_time=None, end_time=None) -> pd.DataFrame:
         df_full = None
         for dl in self.data_loader_l:
-            df_current = dl.load(instruments, start_time, end_time)
+            try:
+                df_current = dl.load(instruments, start_time, end_time)
+            except:
+                instruments = None
```

Comment:
whguWe can use `instruments=None`  intead of changing the variable (which may cause later errors.)